### PR TITLE
Add scql issue template

### PR DESCRIPTION
**Pull request by Chrisdehe**
I have read the CLA Document and I hereby sign the CLA

Add scql issue template

### CODE REVIEW

**Comment by Chrisdehe** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/ISSUE_TEMPLATE/Consulting_issue_template.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        - Feature
+        - Documentation
+        - Others
+    validations:
+      required: true
+  - type: dropdown
+    id: searched-for-existing-issues
+    attributes:
+      label: Have you searched for existing issues? 
+      description: Suggest searching first existing [documents](https://www.secretflow.org.cn/zh-CN/docs/scql/main/) and [issues](https://github.com/secretflow/scql/issues)
```
Comment (jingshi-ant) [inline .github/ISSUE_TEMPLATE/Consulting_issue_template.yaml:26]: searching first这里需要更新

**Comment by jingshi-ant** [comment]
LGTM

**Review by tongke6 — APPROVED** [review]
LGTM

**Pull request merged**